.github/workflows/build.yaml

```diff
@@ -334,6 +334,8 @@ jobs:
       - name: Set up QEMU
         if: matrix.arch != 'amd64'
         uses: docker/setup-qemu-action@v3
+        with:
+          image: public.ecr.aws/eks-distro-build-tooling/binfmt-misc:qemu-v7.0.0
```

Comment:
should we consider putting this into quay instead?

Also, we'd have to remember to update this once `docker/setup-eqmu-action` is updated, right? Can you add a comment in this file to be on the lookout for that? (though, these deps are updated by dependabot, so that may be missed...)